source/core/gridfs.txt

```diff
@@ -16,6 +16,16 @@ GridFS
 that exceed the :term:`BSON`\-document :ref:`size limit
 <limit-bson-document-size>` of 16 MB.
 
+
+
+.. important::
+
+   GridFS does not support multi-document transactions. Transactions are represented in a single oplog 
+   entry and must be under the 16MB limit. Also, transactions default lifetime is 60 seconds. Large files, 
+   like the ones you may be using GridFs, often take too long to upload. Since, GridFS supports large files 
+   above 16 MB this makes it impractical to support multi-document transactions. 
```

Comment:
In 4.2 each operation within a transaction gets its own oplog entry so the 16MB limit is no longer an issue and the txn lifetime is adjustable so I'm not sure exactly where that leaves the surface area of the limitation.

I think it's probably okay to not include all of the explanation, so let's just make this a really simple note:

```rst
.. note::
   
   GridFS does not support :doc:`multi-document transactions </core/transactions>`.
```

Reply:
Done